dotnet/samples/KernelSyntaxExamples/Example76_StronglyTypedKernelResult.cs

```diff
@@ -0,0 +1,148 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Threading.Tasks;
+using Azure.AI.OpenAI;
+using Microsoft.SemanticKernel;
+using System.Text.Json;
+using System.Diagnostics;
+using System.Collections.Generic;
+using Xunit;
+using Xunit.Abstractions;
+using Examples;
+
+// The following example shows how to receive the results from the kernel in a strongly typed object
+// wich stores the usage in tokens and converts the JSON result to a strongly typed object, where a validation can also
+// be performed
+public class Example76_StronglyTypedKernelResult : BaseTest
```

Comment:
Seems like this should be Example77 now :) Also, since we don't have `KernelResult` anymore, I believe we should rename this example to `Example77_StronglyTypedFunctionResult`.

Reply:
Alrighty!